Core/clim-core/presentation-translators.lisp

```diff
@@ -73,11 +73,6 @@ and used to ensure that presentation-translators-caches are up to date.")
   ((translators :accessor translators :initarg :translators
```

Comment:
Commit messagae: "This change render" -> "This change renders"

Reply:
fixed